chat-client/src/client/chat.ts

```diff
@@ -183,8 +195,25 @@ export const createChat = (
                 }
 
                 const allExistingTabs: MynahUITabStoreModel = mynahUi.getAllTabs()
+
+                const additionalContextCommands: any[] = []
+                featureConfig.forEach((featureContext: FeatureContext, _feature: string) => {
+                    additionalContextCommands.push({
```

Comment:
I do not think it's safe to use all the items from featureConfig as additional context commands. Can we just pull that 'highlightCommand' here, which is used for @sage?